event/event.go

```diff
@@ -5,12 +5,38 @@ import (
 	"time"
 )
 
+// Category define how to display the Category.
+type Category int32
+
+const (
```

Comment:
I didn't say change the names, i said add a comment to keep in sync with protobuf.